lib/Wikia/src/Service/Gateway/ConsulUrlProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Wikia\Service\Gateway;
+
+
+use Http;
+
+class ConsulUrlProvider implements UrlProviderInterface {
+
+	private $healthUrlSchema = '{consulUrl}/v1/health/service/{serviceName}?passing&tag={serviceTag}';
```

Comment:
Is there a way to specify whether or not we care about stale entries? The `golang` client provides [this feature](https://github.com/hashicorp/consul/blob/master/api/api.go#L27).